mod/likes/start.php

```diff
@@ -18,6 +18,13 @@ function likes_init() {
 	elgg_register_plugin_hook_handler('register', 'menu:river', 'likes_river_menu_setup', 400);
 	elgg_register_plugin_hook_handler('register', 'menu:entity', 'likes_entity_menu_setup', 400);
 	elgg_register_plugin_hook_handler('permissions_check', 'annotation', 'likes_permissions_check');
+	
+	// need to register for the specific types instead of all as all is evaluated after the specific types
```

Comment:
...So feel free to use `'permissions_check:annotate', 'all'` and why not priority 0.